docs/source/architecture/servicefees.rst

```diff
@@ -0,0 +1,79 @@
+
+Alices are required to escrow a deposit covering the cost of the entire duration of the policy, but fees are paid out to stakers once per period, after their Ursula confirms activity. Therefore the minimum sum of fees a staker will receive each period is the product of their specified minimum fee rate and the number of active sharing policies their Ursula has been assigned.
+
+
+Global fee range
+----------------
+
+The global fee range is a means of establishing quasi-universal pricing for the NuCypher service. It is enforced via the function ``feeRateRange`` in ``PolicyManager.sol``, which specifies constraints, expressed in GWEI, per sharing policy and per 24h period. The minimum fee rate rate chosen by stakers must fall within the global fee range. The network will launch with the following parameters:
+
+**Minimum fee rate**
+
+XX GWEI (~$X.XX x10^-X) *per period*
+
+XX,XXX GWEI ($X.XXX) *per year*
+
+**Maximum fee rate**
+
+X,XXX GWEI (~$X.XX x10^-X) *per period*
+
+XXX,XXX GWEI ($X.XXX) *per year*
+
+**Default fee rate**
+
+XXX GWEI (~$X.XX x10^-X) *per period*
+
+XX,XXX GWEI ($X.XXXX) *per year*
+
+*1 GWEI = 10^-9 ETH*
+
+*USD conversion utilizes the ETHUSD 100-day rolling average of 1 ETH = $222 (08.08.20)*
+
+Note that the minimum and maximum fee rate are an upper and lower bound to constrain the fee rate a staker may offer. The default fee rate is the rate that will be displayed and offered to Alices if the staker chooses not to configure this parameter themselves.
+
+The fee range must be adhered to in identical fashion by all NuCypher stakers, regardless of their stake size or capacity. The fee range applies to all sharing policies, irrespective of the volume of re-encryption requests or other distinguishing attributes. It also applies equally to all periods in the future, until the moment that the global fee range’s parameters are adjusted or the range is removed, via official governance processes (see below). If an update of this sort occurs, sharing policies that were previously established, but have not yet expired, should not have the per-period fee rate retroactively modified.
+
+Governance & pricing paper
+---------------------------------------
+
+In order to successfully execute the ``PolicyManager.sol`` contract, the global fee range must be adhered to by the Ursula (and Alice). Failing this, the contract will throw up an error and it will not be possible to commence a commercial engagement or pay/receive fees. Attempts to circumvent NuCypher’s smart contracts are likely to be futile (except perhaps with limited, existing, trust-heavy relationships), given the requirement of coordinated modification and redeployment by network users and a critical mass of other stakers.
```

Comment:
execute -> interact 
removed `(except perhaps with limited, existing, trust-heavy relationships)` -> should we mention this?
```suggestion
In order to successfully interact with the ``PolicyManager.sol`` contract, the global fee range must be adhered to by the Ursula (and Alice). Failing this, the contract will throw up an error and it will not be possible to commence a commercial engagement or pay/receive fees. Attempts to circumvent NuCypher’s smart contracts are likely to be futile given the requirement of coordinated modification and redeployment by network users and a critical mass of other stakers.
```

Reply:
I was on the fence about this – it's good to acknowledge extra-protocol interactions, rather than pretend they don't exist. At the same time perhaps saying "circumventing is futile" is taking too strong a stance on it. The main point is that you can't commercially engage unless you adhere to the fee range